05-Mining-Protocol.md

```diff
@@ -448,3 +518,57 @@ This message can be sent only to connections that don’t have `REQUIRES_STANDAR
 | ---------------- | ------------- | ----------------------------------------------------------------------------------------- |
 | group_channel_id | U32           | Identifier of the group where the standard channel belongs                                |
 | channel_ids      | SEQ0_64K[U32] | A sequence of opened standard channel IDs, for which the group channel is being redefined |
+
+## 5.5 Proxies
+
+Proxies are entities that aggregate hashrate from multiple Mining Devices into a single Connection.
+
+The following protocol primitives are based on the assumption that proxies are part of the infrastructure:
+- Extended Extranonces: allowing for efficient hashing space distribution.
+- Future Jobs: allowing for in-site caching of Future Jobs (therefore avoiding empty Jobs).
+
+Proxies can be nested into multiple layers, depending on different needs for hasher infrastructure.
+The infrastructure complexity may vary according to operational requirements, but a single proxy layer should suffice for most simple setups.
+
+Proxies can be split in two main categories:
+
+### 5.5.1 Mining Proxy
+
+A Mining Proxy covers the full scope of the Mining Protocol. It supports all types of channels, both for Upstream and Downstream Connections.
+
+|                | **Standard Channels** | **Group Channels** | **Extended Channels** |
+|----------------|-----------------------|--------------------|-----------------------|
+| **Upstream**   | yes                   | yes                | yes                   |
+| **Downstream** | yes                   | yes                | yes                   |
+
+A Mining Proxy is responsible for delivering Jobs to Downstream Channels and is a crucial piece of infrastructure for optimizing the hash space distribution across multiple SV2 Mining Devices (plus SV1 Mining Devices, in case a Translator Proxy is available Downstream).
+
+A Mining Proxy can forward multiple Standard Channels to the Upstream, ideally one representing each SV2 Mining Device Downstream.
+The Upstream (Pool, JDC or another Mining Proxy) is expected to send a `SetGroupChannel` message aggregating different Standard Channels.
```

Comment:
Is the `SetGroupChannel` triggered by some specific behaviour? 
Or the trigger to send it can be arbitrary defined by pools?

Reply:
the main criteria I can think is: all Standard Channels under the same Connection are unified under the same Group Channel

but maybe the spec authors can elaborate on some other scenarios